examples/random-tfjob-example.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: "kubeflow.org/v1alpha1"
+kind: StudyJob
+metadata:
+  namespace: katib
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: random-example
+spec:
+  studyName: random-example
+  owner: crd
+  optimizationtype: maximize
+  objectivevaluename: Validation-accuracy
+  optimizationgoal: 0.99
+  requestcount: 4
+  metricsnames:
+    - accuracy
+  parameterconfigs:
+    - name: --lr
+      parametertype: double
+      feasible:
+        min: "0.01"
+        max: "0.03"
+    - name: --num-layers
+      parametertype: int
+      feasible:
+        min: "2"
+        max: "5"
+    - name: --optimizer
+      parametertype: categorical
+      feasible:
+        list:
+        - sgd
+        - adam
+        - ftrl
+  workerSpec:
+    goTemplate:
+        rawTemplate: |-
+          apiVersion: "kubeflow.org/v1alpha1"
```

Comment:
This should be using the v1beta1 API now. V1apha1 is deprecated.

Reply:
SGTM